code/modules/mob/dead/new_player/new_player.dm

```diff
@@ -399,8 +399,11 @@
 		if(!arrivals_docked)
 			var/atom/movable/screen/splash/Spl = new(character.client, TRUE)
 			Spl.Fade(TRUE)
-			character.playsound_local(get_turf(character), 'sound/voice/ApproachingTG.ogg', 25)
-
+			switch(rand(1, 100))
+				if(1 to 99)
+					character.playsound_local(get_turf(character), 'sound/voice/ApproachingYOG.ogg', 25)
+				if(100)
+					character.playsound_local(get_turf(character), 'sound/voice/ApproachingYOGbutepic.ogg', 25)
```

Comment:
sorta weird way of doing this, but i guess it works

Reply:
sound updated as of next push